### DISCUSSION TITLE
ssl in async in micropython 1.21

### DISCUSSION THREAD

**Discussion by sophiedegran** [miguelgrinberg/microdot#174] [General]
Miguel many thanks for your fantastic framework.
It's seem ssl async is now implemented in micropython 1.21 did we have to do something to used it ?
-T

**Comment by miguelgrinberg** [comment]
I don't see any mention of this in the release notes. Where is this new feature documented?

**Reply by sophiedegran** [reply]
Yes it's very strange... Jimmo tell about that in discussion 12222 on micropython site... I will see with that and tell you..
-T

**Reply by sophiedegran** [reply]
Ok dpgeorge push it to release 1.22 ... (see pull 11897) they work on it ☺ that wil be a great improvement!
